Templates/GitHubCommenter-FSharp/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "defaultFunctionName": "GitHubCommenterFSharp",
+    "description": "$FSharp_description",
```

Comment:
should be GitHubCommenterFSharp_description
Also should be add to resource file